### PR TITLE
Initial db setup

### PR DESCRIPTION
**Pull request by IrinaGM**
* setup the DB tables creation - please see the update ERD as well: https://www.figma.com/file/3nFdLudVFuNg3e06mFv8WJ/Resource-Wall-Whiteboard?type=whiteboard&node-id=64-3&t=z9hPVJhfwFg5ZMsR-0
* setup the seeds.
* removed widgets as not required for our project.

To get the latest changes of the DB please don't forget to run `npm run db:reset` after you pull to you local main branch, and before you branch again for a feature.

### CODE REVIEW

File: db/seeds/02_topics.sql
Comment (SowmyaVijayendra) [inline db/seeds/02_topics.sql:1]: Are we not storing hashed passwords in the table?

File: db/seeds/05_ratings.sql
Comment (SowmyaVijayendra) [inline db/seeds/05_ratings.sql:1]: I loved the comments :)

File: db/seeds/02_topics.sql
Comment (IrinaGM) [reply db/seeds/02_topics.sql:1]: Let's keep it as a stretch

File: db/seeds/05_ratings.sql
Comment (IrinaGM) [reply db/seeds/05_ratings.sql:1]: Thanks!

**Review by SowmyaVijayendra — APPROVED** [review]
approved

**Pull request merged**